istio/supported_ops.go

```diff
@@ -34,34 +37,42 @@ const (
 
 var supportedOps = map[string]supportedOperation{
 	installIstioCommand: {
-		name: "Install the latest version of Istio",
+		name: "Latest Istio without mTLS",
 		// templateName: "install_istio.tmpl",
+		opType: meshes.OpCategory_INSTALL,
 	},
 	installmTLSIstioCommand: {
-		name: "Install the latest version of Istio with mTLS",
+		name:   "Latest Istio with mTLS",
+		opType: meshes.OpCategory_INSTALL,
 	},
 	installBookInfoCommand: {
-		name: "Install the canonical Book Info Application",
+		name: "Canonical Book Info Application",
```

Comment:
Perhaps, displace "canonical" with "sample".

Reply:
I can drop `canonical`. But we probably don't want `sample` because it already belongs to sample application category.